src/testUtils/mocks/dataSources.ts

```diff
@@ -950,7 +952,7 @@ const createMockCodec = (codec: object, isEmpty: boolean): Codec => {
 /**
  * @hidden
  */
-const createMockStringCodec = (value?: string): Codec =>
+export const createMockStringCodec = (value?: string): Codec =>
```

Comment:
This shouldn't be exported. You should create a `createMockTextCodec` that uses this and export that